tensorizer/tensorizer.py

```diff
@@ -529,7 +605,8 @@ def write_tensor(self, idx, name, tensor_type, tensor) -> None:
         else:
             comp_report = ""
         logger.info(
-            f"{idx}:{typ}:{name} - {tensor.shape} -> {ds_bytes}{comp_report}"
+            f"{idx}:{typ}:{name} - {str(dtype_bytes)} - "
```

Comment:
Bytes-to-string conversion should be done with `dtype_bytes.decode('utf-8')`; calling `str` only gives a representation of the bytes object, like `b'Content'`.

For comparison:
```py
>>> b = "Hello ✨".encode("utf-8")
>>> b
b'Hello \xe2\x9c\xa8'
>>> str(b)
"b'Hello \\xe2\\x9c\\xa8'"
>>> b.decode("utf-8")
'Hello ✨'
```

Reply:
Fixed. :+1: